manifests/v1beta1/components/webhook/secret.yaml

```diff
@@ -0,0 +1,5 @@
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: katib-webhook-cert
```

Comment:
Do we know if [cert manager `Certificate` is going](https://github.com/kubeflow/katib/blob/eac65bf8e31fe4d3a2c4dbb20be30d600c086157/manifests/v1beta1/installs/katib-cert-manager/certificate.yaml#L15) to patch the existing Secret ?

Reply:
Yes, the cert-manager will inject certs to the existing secret.
I confirmed the behavior using cert-manager v1.12.0.